src/lesson7/javaBeginnerChapter2/task27/CreditCard.java

```diff
@@ -0,0 +1,7 @@
+package lesson7.javaBeginnerChapter2.task27;
+
+abstract class CreditCard extends BankCard {
+    public CreditCard(String name) {
```

Comment:
The same as previous comment - you should extend superclass, but not duplicate it

Reply:
fixed